CHANGELOG.md

```diff
@@ -46,6 +47,7 @@ Please add to relevant subsections here on every PR where this is applicable.
 ### Fixed
 
 * In case of transient server issues when creating the wandb API key kubernetes secret, we'll retry up to 5 times by @TimH98 in https://github.com/wandb/wandb/pull/7108
+* Fix handling of saving training files to Artifacts in the OpenAI Fine-Tuning integration by @morganmcg1 in https://github.com/wandb/wandb/pull/7150
```

Comment:
this one should be in the unreleased section, could you please fix this? @morganmcg1